lib/util/native/tray_helper.dart

```diff
@@ -14,31 +15,59 @@ Future<void> initTray() async {
   if (!checkPlatformHasTray()) {
     return;
   }
-  try {
-    if (checkPlatform([TargetPlatform.windows])) {
-      await trayManager.setIcon(
-        Assets.img.logo32Ico,
-      );
-    } else if (checkPlatform([TargetPlatform.macOS])) {
-      await trayManager.setIcon(Assets.img.logo32BlackWhite.path, isTemplate: true);
-    } else {
-      await trayManager.setIcon(Assets.img.logo32Png.path);
+  if (!checkPlatform([TargetPlatform.linux])) {
+    try {
+      if (checkPlatform([TargetPlatform.windows])) {
+        await tm.trayManager.setIcon(
+          Assets.img.logo32Ico,
+        );
+      } else if (checkPlatform([TargetPlatform.macOS])) {
+        await tm.trayManager.setIcon(Assets.img.logo32BlackWhite.path, isTemplate: true);
+      } else {
+        await tm.trayManager.setIcon(Assets.img.logo32Png.path);
+      }
+
+      final items = [
+        tm.MenuItem(
+          key: TrayEntry.open.name,
+          label: t.tray.open,
+        ),
+        tm.MenuItem(
+          key: TrayEntry.close.name,
+          label: t.tray.close,
+        ),
+      ];
+      await tm.trayManager.setContextMenu(tm.Menu(items: items));
+      await tm.trayManager.setToolTip(t.appName);
+    } catch (e) {
+      print(e);
     }
+  } else {
+    final SystemTray systemTray = SystemTray();
+
+    await systemTray.initSystemTray(
+      title: t.appName,
+      iconPath: Assets.img.logo32Png.path,
+    );
+
+    final Menu menu = Menu();
+    await menu.buildFrom([
+      MenuItemLabel(label: t.tray.open, onClicked: (menuItem) async {
+        await windowManager.show();
+        await windowManager.focus();
+    }),
+      MenuItemLabel(label: t.tray.close, onClicked: (menuItem) => windowManager.destroy()),
```

Comment:
You can use `exit(0)` for that so we don't rely on a library.

Reply:
Done